src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/AC5052A.scala

```diff
@@ -27,11 +27,49 @@ case class AC5052A(value: Option[Int]) extends CtBoxIdentifier(name = "Debtors d
 
 with Validators {
 
+  private def noteHasValue(boxRetriever: Frs102AccountsBoxRetriever): Boolean = {
+    boxRetriever match {
+      case x: AbridgedAccountsBoxRetriever =>
+        x.ac5052A().hasValue ||
+          x.ac5052B().hasValue ||
+          x.ac5052C().hasValue
+
+      case x: FullAccountsBoxRetriever =>
+        x.ac134().hasValue ||
+          x.ac135().hasValue ||
+          x.ac138().hasValue ||
+          x.ac139().hasValue ||
+          x.ac136().hasValue ||
+          x.ac137().hasValue ||
+          x.ac140().hasValue ||
+          x.ac141().hasValue ||
+          x.ac5052A().hasValue ||
+          x.ac5052B().hasValue ||
+          x.ac5052C().hasValue
+    }
+  }
+
   override def validate(boxRetriever: Frs102AccountsBoxRetriever): Set[CtValidation] = {
     collectErrors (
-      cannotExistIf(value.nonEmpty && boxRetriever.ac52.noValue),
+      failIf(boxRetriever.ac52().noValue)(validateCannotExist(boxRetriever)),
```

Comment:
You are allowed to also have the note if the previous year value exists as far as I know